Azure/azure-cli
# {Network} az network application-gateway frontend-ip update: Deprecating the --public-ip-address parameter (#13891)

az network application-gateway frontend-ip update: Deprecating the --public-ip-address parameter (#13891)

Fixing punctuation in help message for `az network application-gateway address-pool update`

**Description**  
az network application-gateway frontend-ip update does not work for public ip addresses. The issue is documented in https://github.com/Azure/azure-cli/issues/13891. The argument needs to be deprecated to avoid confusion. This has been discussed with @MyronFanQiu 

**Testing Guide**  
az cli examples of the argument not working are in https://github.com/Azure/azure-cli/issues/13891

This PR has been discussed with @MyronFanQiu and tagging him as the reviewer

The documentation for this also needs to reflect this change: https://docs.microsoft.com/en-us/cli/azure/network/application-gateway/frontend-ip?view=azure-cli-latest#az-network-application-gateway-frontend-ip-update. Not sure how that is done.

This checklist is used to make sure that common guidelines for a pull request are followed.
